crates/bevy_core_pipeline/src/deferred/mod.rs

```diff
@@ -0,0 +1,102 @@
+pub mod node;
+
+use std::cmp::Reverse;
+
+use bevy_ecs::prelude::*;
+use bevy_render::{
+    render_phase::{CachedRenderPipelinePhaseItem, DrawFunctionId, PhaseItem},
+    render_resource::{CachedRenderPipelineId, TextureFormat},
+};
+use bevy_utils::FloatOrd;
+
+pub const DEFERRED_PREPASS_FORMAT: TextureFormat = TextureFormat::Rgba32Uint;
+
+/// Opaque phase of the 3D Deferred pass.
+///
+/// Sorted front-to-back by the z-distance in front of the camera.
```

Comment:
This might end up needing to be changed depending on @superdump's performance work, if sorting by PSO ends up faster than z-sorting.

Reply:
I can update the comment if we do that. I just copied it directly from the prepass `mod.rs`. Deferred doesn't care about the order or sorting (nor do I) so it's just the comment that'll need addressing. (not to imply you didn't already know that, just mentioning it for clarity)